java-checks/src/main/java/org/sonar/java/checks/CommentedOutCodeLineCheck.java

```diff
@@ -114,6 +118,12 @@ public AnalyzerMessage collectIssues(List<AnalyzerMessage> issues, SyntaxTrivia
     return issue;
   }
 
+  private boolean isLicenseOrNaturalLanguage(String line) {
+    String trimmedLine = line.trim();
```

Comment:
- why doing a `.trim()` before applying regular expressions that start and end with `.*` or`\s*`?
- why complexifying `NATURAL_LANGUAGE_PATTERN` with a `$` at the end, if it is only used with `.matches()` 
- why using regular expression starting and ending with `.*` and trying of match the full line `.matcher(trimmedLine).matches()` instead of removing the useless `.*` and stoping the evaluation as soon as we found what we want `.matcher(line).find()`